metals/src/main/scala/scala/meta/internal/metals/WorkspaceSymbolProvider.scala

```diff
@@ -31,14 +31,28 @@ final class WorkspaceSymbolProvider(
   var inDependencies: ClasspathSearch =
     ClasspathSearch.fromClasspath(Nil, bucketSize)
 
-  def search(query: String): Seq[l.SymbolInformation] = {
-    search(query, () => ())
+  def search(
```

Comment:
Are these changes used?

Reply:
Not after switching to the `DefinitionProvider`, no. Thanks for catching - reverted.